sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -316,6 +316,203 @@ ALTER TABLE `form_care_plan` CHANGE `date` `date` DATETIME NULL DEFAULT NULL;
 ALTER TABLE api_token ADD COLUMN `context` TEXT COMMENT 'context values that change/govern how access token are used';
 #EndIf
 
+#IfRow2D list_options list_id language notes eng
+CREATE TEMPORARY TABLE lang_updates_610 (notes char(15), twodigit_notes char(2));
+START TRANSACTION;
+INSERT INTO lang_updates_610 VALUES
+('abk','ab'),
+('aar','aa'),
```

Comment:
@bradymiller @adunsulag I had to fix conflicts so you guys need to ensure sql upgrade and database are still intact.
Looks okay to me but check me anyways. Thanks